docs/source/data_acquisition/DataProviders.rst

```diff
@@ -386,25 +386,36 @@ Connecting to an OData Source
 -----------------------------
 :py:mod:`OData driver API documentation<msticpy.data.drivers.odata_driver>`
 
-You can also connect to OData based data sources such as the MDATP API,
-or the Security Graph API. These connections often rely on having a
-dedicated Azure AD app for handling the authentication process.
+You can also connect to OData based data sources such as the Microsoft Defender,
```

Comment:
Did you mean Microsoft 365 Defender here? I think Microsoft Defender is the proper name for Defender AV